projects/parsers/example.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan 25 06:55:48 2019
+
+@author: ed
+"""
+
+from loop_report import LoopReport
+import pandas as pd
+import json
+
+# %% file name and path
+file_path = "/Users/ed/Desktop/"
+file_name = "Loop Report 2019-01-16 08-39-03-06-00"
```

Comment:
Can you replace this line and the line above with the example file that we agreed to post to github? Also, we should use os.path.join rather than hard coding (and yes, I know I was the one to include this line of code)

Reply:
should be something like this: 

import os

# %% file name and path
file_path = os.path.join("..", "tests", "parsers", "files")
file_name = "LoopReport"